rst_files/getting_started.rst

```diff
@@ -224,41 +214,25 @@ Now try copying the following into a notebook cell and hit ``Shift-Enter``
     gr(fmt=:png)
     plot(sin, -2π, 2π, label="sin(x)")
 
-You'll see something like this (although the style of plot depends on your
-installation --- more on this later)
 
 **Note**: The "time-to-first-plot" in Julia takes a while, since it needs to precompile everything
 
-.. figure:: /_static/figures/nb4_julia.png
-   :scale: 100%
-
-
-Working with the Notebook
------------------------------
-
-Let's go over some more Jupyter notebook features --- enough so that we can press ahead with programming
-
 
 Tab Completion
 ^^^^^^^^^^^^^^^^^^
 
-A simple but useful feature of `IJulia <https://github.com/JuliaLang/IJulia.jl>`_ 
-(the Jupyter kernel for Julia) is tab completion
+Tab completion in Jupyter makes it easy to find Julia commands and functions available
```

Comment:
I'm hesitant to say `IJulia` because it can be confusing for a beginner, the subtle difference between julia, IJulia, and Jupyter. I think it's fine to just say Jupyter because if you're using julia in Jupyter, you're using IJulia.